### DISCUSSION TITLE
Is this for private torrent and how to config cross_seed in a docker

### DISCUSSION THREAD

**Discussion by avatasia** [StuffAnThings/qbit_manage#463] [General]
I knew that cross_seed is a docker command ,and many configurations with jackett.
But I can't find this in qbit_manage.
So how to cross seed with qbit_manage.

**Comment by bakerboy448** [comment]
Qbit manage does not cross seed. Please read the wiki.

https://www.cross-seed.org/ is likely what you're looking for
